src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Dsa.cs

```diff
@@ -65,7 +65,7 @@ internal static int DsaKeySize(SafeDsaHandle dsa)
             // there were some leading zero bytes truncated due to BIGNUM semantics.
             // This minimizes the change the key size is wrong due to leading zero bits; there is only
             // a 2^64 chance that the key has 64 leading zero bits and thus the keysize will be wrong.
```

Comment:
@steveharter Yeah, I understand why we add 7, divide by 8, and multiply by 8.  The comment says there's a 1:2^64 chance that there are 64 leading zeroes; but that's not particularly relevant.

> p = a prime modulus, where 2<sup>L-1</sup> &lt; p &lt; 2<sup>L</sup> for 512 <=  L <= 1024 and L a multiple of 64

(FIPS 186-2, section 4)

Technically, BN_num_bits should always just be correct for `p` (or, rather `L`, since `L` is the size of `p` in bits); unless it's always off by 1 for indicating the significance of the "not negative" padding bit/byte.  `q` appears to be equally rigid, but `g`, `x` and `y` are soft.  Those relationships seem to also hold under FIPS 186-3.

That said, I'm not opposed to byte-alignment-normalization of the value for just-in-case (defense in depth for a future enhancement to the DSA spec); just that the *comment* seems to be a bit off.